### ISSUE TITLE
Provide values for the input features along with the predictions

### ISSUE THREAD

**Issue by ragesoss** [wiki-ai/ores#100]
When I get the wp10 score for a revision, what I'd really like to be able to do is to get the values for the features that model uses to calculate the wp10 prediction.

Ideally, these feature scores would be as close to the source text as possible, ie, actual number of references rather than log(number of references) or similar.

Having these feature scores would allow my project to start figuring out how to provide specific advice for article improvement based on which features scores are particularly weak relative to the others.

**Comment by halfak** [comment]
Good idea.  I [added a trim() function](https://github.com/wiki-ai/revscoring/pull/203) to revscoring.  That function will "trim" a feature dependency tree down to its *generated features* (no modifiers like [`log()`](http://pythonhosted.org/revscoring/revscoring.features.html#revscoring.features.modifiers.log) and no constants).  These features can be extracted independently and provided as `cache` to a call to [`solve()`](http://pythonhosted.org/revscoring/revscoring.dependencies.html#revscoring.dependencies.functions.solve) to extract the final feature values to provide to the model.

We'll need some way to provide feature values to a call to ORES.  Right now, feature instanaces themselves generate consistent hashes by using names (e.g. 'features.diff.bytes_changed').  This allows new feature instances to replaces others in-context.  It seems like we could use this same naming strategy to populate the cache from an API call.  

So, `scores/enwiki/damaging/2345675?languages.english.badwords_added=5` would run the exact same computation except that it would replace the `revscoring.languages.english.badwords_added` feature with `5`.  For these calls, the cache would need to be disabled.  We'd also need to add an argument to the call that would output the raw generated feature values.  I'm liking just having a boolean param called "?features".  This would result in including a "features" object within the score JSON.  This features object should simply be the same dict that we might pass as `cache` to `solve()` -- *generated feature* name-value pairs.
